pod-security/restricted/disallow-capabilities-strict/disallow-capabilities-strict.yaml

```diff
@@ -38,7 +38,7 @@ spec:
                 all:
                 - key: ALL
                   operator: AnyNotIn
-                  value: "{{ element.securityContext.capabilities.drop || '' }}"
+                  value: "{{ element.securityContext.capabilities.drop[] || `[]` }}"
```

Comment:
There was a to_upper in the other PR pankaj put: https://github.com/nirmata-add-ons/policies/pull/11/files

`value: "{{ element.securityContext.capabilities.drop[].to_upper(@) || `[]` }}"`

Is it not needed?

Reply:
@pns-nirmata 
to_upper is removed because if [this](https://github.com/nirmata-add-ons/policies/pull/11#issuecomment-1368464052) comment in the other PR